gslab_fill/tests/test_tablefill.py

```diff
@@ -23,19 +23,43 @@ def setUp(self):
             os.mkdir('./build/')
 
     def testInput(self):
-        with nostderrout():
-            message = tablefill(input    = '../../gslab_fill/tests/input/tables_appendix.txt ' + \
-                                           '../../gslab_fill/tests/input/tables_appendix_two.txt', 
-                                template = '../../gslab_fill/tests/input/tablefill_template.lyx', 
-                                output   = './build/tablefill_template_filled.lyx')
-        self.assertIn('filled successfully', message)                        
-        tag_data = open('../../gslab_fill/tests/input/tablefill_template.lyx', 'rU').readlines()
-        filled_data = open('./build/tablefill_template_filled.lyx', 'rU').readlines()
-        self.assertEqual(len(tag_data), len(filled_data))
-        for n in range(len(tag_data)):
-            self.tag_compare(tag_data[n], filled_data[n]) 
-    
-    def tag_compare(self, tag_line, filled_line):
+        for ext in ['lyx', 'tex']:

```

Comment:
@yuchuan2016 , how do you feel about adding comments for these functions? I know we seem to not have comments for any of the tests (and this is from old code) but this is gonna get increasingly difficult to maintain, and since you know them pretty well right now this may be an opportune time to do so.